src/OpenDebugAD7/LanguageUtilities.cs

```diff
@@ -0,0 +1,53 @@
+﻿// // Copyright (c) Microsoft. All rights reserved.
+// // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+
+namespace OpenDebugAD7
+{
+    internal static class LanguageUtilities
+    {
+        /// <summary>
+        /// This method does a simple validation on the input 'identifier'.
+        /// </summary>
+        /// <param name="identifier">the identifier to validate</param>
+        /// <returns>'true' if this is a valid identifier name. 'false' if otherwise.</returns>
+        public static bool IsValidIdentifier(string identifier)
+        {
+            // Check to see if identifier is null or blank.
+            if (string.IsNullOrWhiteSpace(identifier))
+            {
+                return false;
+            }
+
+            string[] parts = identifier.Split();
```

Comment:
>string[] parts = identifier.Split(); [](http://example.com/codeflow?start=12&length=36)

Is there a reason you are splitting this?

Reply:
I was going to do something more specific but I think I'm good with just validating first character and equal comparisons.